src/main/kotlin/baseball/Computer.kt

```diff
@@ -0,0 +1,31 @@
+package baseball
+
+import baseball.C.RANDOM_SIZE
+import camp.nextstep.edu.missionutils.Randoms
+
+class Computer {
+    private var randomNumList: MutableList<String> = mutableListOf()
+
+    //난수를 생성하는 함수
+    fun makeRandomNumList() {
+        var temp = ""
+        randomNumList = mutableListOf<String>()
+        while (randomNumList.size < RANDOM_SIZE) {
+            temp = Randoms.pickNumberInRange(1, 9).toString()
+            if (!randomNumList.contains(temp)) {
+                randomNumList.add(temp)
+            }
+        }
+    }
+
+    //게임 결과를 도출하는 함수
+    fun getGameResult(userInputList: List<String>): Pair<Int, Int> {
+        var (strike,ball) = Pair(0,0)
+        for (i in userInputList.indices) {
+            if (userInputList[i] == randomNumList[i]) strike++
+            else if ((userInputList[i] != randomNumList[i]) && randomNumList.contains(userInputList[i])) ball++
```

Comment:
뭔가 설명이 잘 안되는데
이미 if 에서 조건을 검사해서 else if 에서 똑같은 조건을 확인할 필요가 없는 것 같습니다.
```suggestion
            if (userInputList[i] == randomNumList[i]) strike++
            else if (randomNumList.contains(userInputList[i])) ball++
```